src/app/Steinpilz.DevFlow.Fake.Lib/lib.fs

```diff
@@ -40,7 +40,11 @@ type BuildParams = {
     VersionSuffix: string
 
     NuGetFeed: NuGetFeed
+    NuGetSymbolFeed: NuGetFeed

```

Comment:
Let's deal with the separate symbol packages and feeds in the scope of another issue when it's needed. Please move `NuGetSymbolFeed` and related code to a private branch `vm/nuget-symbol-feed`.